README.md

```diff
@@ -34,14 +34,26 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML are assets for which its attributes are described within the code. 
+"div" is not semantic.
+
```

Comment:
That is correct, also accessibility in the job world would be a major reason -> https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML